src/api/app/models/bs_request.rb

```diff
@@ -297,6 +297,10 @@ def set_ignore_delegate
     @ignore_delegate = true
   end
 
+  def enforce_branching
+    @enforce_branching = true
```

Comment:
```
@enforce_branching ||= true
```

Reply:
hmm, in what state of @enforce_branching would make your suggestion a difference to the resutl?